readthedocs/oauth/models.py

```diff
@@ -164,27 +174,27 @@ class RemoteRepository(models.Model):
         null=True,
         blank=True,
     )
-
-    json = models.TextField(_('Serialized API response'))
+    remote_id = models.CharField(
+        db_index=True,
+        max_length=128
+    )
+    vcs_provider = models.CharField(
+        _('VCS provider'),
+        choices=VCS_PROVIDER_CHOICES,
+        max_length=32
+    )
 
     objects = RemoteRepositoryQuerySet.as_manager()
 
     class Meta:
-        ordering = ['organization__name', 'name']
+        ordering = ['organization__name', 'full_name']
```

Comment:
I wonder if having a FK lookup in the ordering query here makes it slower. Do we always want this, or only when doing the import form? If only on import, we should probably just force ordering there.